docs/sources/tutorials/get-started.md

```diff
@@ -71,17 +87,169 @@ services:
+1. It provides extra attributes and options, in this case, you will tail log files from the end and not ingest the entire past history.
+
+### Third Component: Write Logs to Loki
+
+Paste this component last in your configuration file:
+
+```alloy
+loki.write "grafana_loki" {
+  endpoint {
+    url = "http://localhost:3100/loki/api/v1/push"
+
+    // basic_auth {
+    //  username = "admin"
+    //  password = "admin"
+    // }
+  }
+}
+```
+
+This last component creates a [loki.write][] component named `grafana_loki` that points to `http://localhost:3100/loki/api/v1/push`.
+This completes the simple configuration pipeline.
+
+{{< admonition type="tip" >}}
+The `basic_auth` is commented out because the local `docker compose` stack doesn't require it. 
+It is included in this example to show how you can configure authorization for other environments.
+For further authorization options, refer to the [loki.write][] component reference.
+
+{{< /admonition >}}
+
+This connects directly to the Loki instance running in the Docker container.
+
+## Reload the Configuration
+
+Copy your local `config.alloy` file into the default configuration file location, which depends on your OS.
+
+{{< code >}}
+
+```macos
+sudo cp config.alloy $(brew --prefix)/etc/alloy/config.alloy
+```
+
+```linux
+sudo cp config.alloy /etc/alloy/config.alloy
+```
+
+{{< /code >}}
+
+Finally, call the reload endpoint to alert {{< param "PRODUCT_NAME" >}} to the configuration change without the need
+for restarting the system service.
+
+```bash
+    curl -X POST http://localhost:12345/-/reload
+```
+
+{{< admonition type="tip" >}}
+This step uses the Alloy UI, which is exposed on `localhost` port `12345`.
+If you chose to run Alloy in a Docker container, make sure you use the `--server.http.listen-addr=0.0.0.0:12345` argument.
+If you don’t use this argument, the [debugging UI][debug] won’t be available outside of the Docker container.
+
+[debug]: ../../tasks/debug/#alloy-ui
+{{< /admonition >}}
+
+The alternative to using this endpoint is to reload the {{< param "PRODUCT_NAME" >}} configuration, which can
+be done as follows:
+
+{{< code >}}
+
+```macos
+brew services restart alloy
+```
+
+```linux
+sudo systemctl reload alloy
+```
+
+{{< /code >}}
+
+## Inspect your Configuration in the {{< param "PRODUCT_NAME" >}} UI
+
+Open [http://localhost:12345] and click the Graph tab at the top.
+The graph should look similar to the following:
+
+{{< figure src="/media/docs/alloy/tutorial/healthy-config.png" alt="Logs reported by Alloy in Grafana" >}}
+
+The UI allows you to see a visual representation of the pipeline you built with your {{< param "PRODUCT_NAME" >}} component configuration.
+We can see that the components are healthy, and you are ready to go.
+
+## Log into Grafana and Explore Loki Logs
+
+Open [http://localhost:3000/explore] to access Explore feature in Grafana.
```

Comment:
Markdown tip: you can use `<http://localhost:3000/explore>` to create a link and avoid needing to add the link definition below.